src/common/meta/src/rpc/ddl.rs

```diff
@@ -92,19 +106,17 @@ impl TryFrom<SubmitDdlTaskRequest> for PbSubmitDdlTaskRequest {
 
 pub struct SubmitDdlTaskResponse {
     pub key: Vec<u8>,
-    pub table_id: TableId,
+    pub table_id: Option<TableId>,
```

Comment:
Why did we change this type to `Option`? Looks unnecessary.

Reply:
Currently, It only returns `table_id` when creating a table. For altering and dropping is no need to return the `table_id`.